### PR TITLE
Update to incoming grpc changes

### PR DESCRIPTION
**Pull request by carsonfarmer**
Signed-off-by: Carson Farmer <carson.farmer@gmail.com>

## Description

Simple update to latest grpc libraries. This should remain WIP until `@textile/thread-net-grpc` is released.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

All standard tests have been run and pass locally. Once the updated library is available, the CI can also be updated.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x]  I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/service-client/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "@improbable-eng/grpc-web": "^0.12.0",
     "@textile/threads-core": "^0.0.3-alpha.0",
     "@textile/threads-encoding": "^0.0.3-alpha.0",
-    "@textile/threads-service-grpc": "^0.1.6",
+    "@textile/thread-net-grpc": "0.0.0",
```
Comment (carsonfarmer) [inline packages/service-client/package.json]: This will have to be updated once the package is released.

**Comment by carsonfarmer** [comment]
This should bring these base modules up to date with https://github.com/textileio/go-threads/pull/272

**Comment by carsonfarmer** [comment]
The test failure is just a timeout, so I can fix that on the next PR (I'll just extend it a bit).

**Comment by carsonfarmer** [comment]
There we go, tests passing again.

File: packages/core/src/thread/key.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+export const keyToString = (k: Buffer) => {
+  return multibase.encode('base32', k).toString()
+}
+
+/**
+ * Key is a thread encryption key with two components.
+ * @param sk Service key is used to encrypt outer log record linkages.
+ * @param rk Read key is used to encrypt inner record events.
+ */
+export class Key {
```
Comment (sanderpick) [inline packages/core/src/thread/key.ts]: Boom!

**Pull request merged**